install/basic/index.rst

```diff
@@ -512,3 +514,351 @@ Whenever you change someting on :guilabel:`.env` file, you will need to rebuild
   docker-compose up -d django
   docker-compose restart geonode
 
+Further Production Enhancements
+===============================
+
+GeoServer Production Settings
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+JVM Settings: Memory And GeoServer Options
+..........................................
+
+The :guilabel:`.env` file provides a way to customize GeoServer JVM Options.
+
+The variable ``GEOSERVER_JAVA_OPTS`` allows you to tune-up the GeoServer container and to enable specific GeoServer options.
+
+.. code-block:: shell
+
+    GEOSERVER_JAVA_OPTS=
+        -Djava.awt.headless=true -Xms2G -Xmx4G -XX:PerfDataSamplingInterval=500 
+        -XX:SoftRefLRUPolicyMSPerMB=36000 -XX:-UseGCOverheadLimit -XX:+UseConcMarkSweepGC 
+        -XX:+UseParNewGC -XX:ParallelGCThreads=4 -Dfile.encoding=UTF8 -Djavax.servlet.request.encoding=UTF-8 
+        -Djavax.servlet.response.encoding=UTF-8 -Duser.timezone=GMT 
+        -Dorg.geotools.shapefile.datetime=true -DGEOSERVER_CSRF_DISABLED=true
+
+``-Djava.awt.headless (true)``
+
+Work with graphics-based applications in Java without an actual display, keyboard, or mouse
+A typical use case of UI components running in a headless environment could be an image converter app. Though it needs graphics data for image processing, a display is not really necessary. The app could be run on a server and converted files saved or sent over the network to another machine for display.
+
+``-Xms2G -Xmx4G``
+
+This means that your JVM will be started with Xms amount of memory and will be able to use a maximum of Xmx amount of memory. Above will start a JVM like with 2 GB of memory and will allow the process to use up to 4 GB of memory. You need to adjust this value depening on your availabnle RAM.
+
+``-DGEOSERVER_CSRF_DISABLED (True)``
+
+The GeoServer web admin employs a CSRF (Cross-Site Request Forgery) protection filter that will block any form submissions that didn’t appear to originate from GeoServer. This can sometimes cause problems for certain proxy configurations. You can disable the CSRF filter by setting the GEOSERVER_CSRF_DISABLED property to true.
+https://docs.geoserver.org/stable/en/user/security/webadmin/csrf.html
+
+
+Whenever you need to change one of more of the JVM options, you will need to restart the GeoServer Docker container.
```

Comment:
Whenever you need to change some of the other available JVM options, you will need to restart the GeoServer Docker container.

?

Reply:
Typo "one or more"